bluepysnap/spike_report.py

```diff
@@ -180,7 +227,7 @@ def _spike_reader(self):
     @cached_property
     def population_names(self):
         """Returns the population names included in this report."""
-        return self._spike_reader.get_populations_names()
+        return sorted(self._spike_reader.get_populations_names())
```

Comment:
Should be reflected in unit tests (I'll check).

Reply:
done here :
```
    def test_iter(self):
        assert sorted(list(self.test_obj)) == ["default", "default2"]
        for spikes in self.test_obj:
            isinstance(spikes, test_module.PopulationSpikeReport)
```